autopilot/src/autopilot.cpp

```diff
@@ -133,6 +132,19 @@ void AutoPilotNode::Start()
 {
   assert(!m_thread);
   autopilotEnabled = true;
+  //  set thruster 0 RPM
+  thruster_control::SetRPM setInitialRPM;
+  setInitialRPM.commanded_rpms = 0.0;
+  thrusterPub.publish(setInitialRPM);
+  //  Set fin to surface
+  double maxCtrlFinAngleRadians = degreesToRadians(maxCtrlFinAngle);
+  fin_control::SetAngles setInitialAnglesMsg;
+  setInitialAnglesMsg.f1_angle_in_radians = maxCtrlFinAngleRadians;
+  setInitialAnglesMsg.f2_angle_in_radians = maxCtrlFinAngleRadians;
+  setInitialAnglesMsg.f3_angle_in_radians = -maxCtrlFinAngleRadians;
+  setInitialAnglesMsg.f4_angle_in_radians = -maxCtrlFinAngleRadians;
+  finsControlPub.publish(setInitialAnglesMsg);
```

Comment:
@patapufate IIUC, the control loop in `workerFunc()` will enforce the current setpoints. If so, why not simply initializing those setpoints to correct values and letting the control loop do the rest?

Reply:
Yes. You're right.